client/src/pages/teamselection/TeamSelectionTable.tsx

```diff
@@ -1,49 +1,123 @@
-import DataTable, { TableColumn } from 'react-data-table-component';
-import RiderLink from '../../components/shared/RiderLink';
-import { RiderParticipation } from '../../models/RiderParticipation';
+import DataTable, { TableColumn } from "react-data-table-component";
+import RiderLink from "../../components/shared/RiderLink";
+import { RiderParticipation } from "../../models/RiderParticipation";
+import { useEffect } from "react";
 
+const conditionalRowStyles = [
+    {
+        when: (row: RiderParticipation) => row.riderId === 0,
```

Comment:
riderId == 0 zou error moeten zijn?
Geen style.

Reply:
Het is een hacky oplossing voor het displayen van de lege regels in je team